apps/meteor/client/views/omnichannel/analytics/AnalyticsPage.tsx

```diff
@@ -31,19 +40,19 @@ const useOptions = (type) => {
 const AnalyticsPage = () => {
 	const t = useTranslation();
 	const [type, setType] = useState('Conversations');
-	const [department, setDepartment] = useState(null);
-	const [dateRange, setDateRange] = useState({ start: null, end: null });
-	const [chartName, setChartName] = useState();
+	const [department, setDepartment] = useState<DepartmentType>('');
```

Comment:
Can this default change affect something? From null to empty?

Reply:
Hey Kevin I remember I checked that the application is handling these values as string types. And when I changed for the default empty string I tested and I didn't see anything weird or different. Do you think this is enough to assume that this change isn't affecting anything?